README.md

```diff
@@ -63,7 +76,51 @@ mackup restore
 
 Done!
 
-You can find more detailed instructions in [INSTALL.md](INSTALL.md).
+You can find more detailed installation instructions in [INSTALL.md](INSTALL.md).
+
+**Scenario 2:** You want to push configuration settings from **some** applications
+from one machine to others for syncing.
+
+Let's say you want to share just your bash configuration files with
+your other machines. Simply create a file in your home directory
+named `.mackup.cfg` and add the following two lines to it with your
+favorite text editor:
+
+```
+[applications_to_sync]
+bash
+```
+
+Save the file and then run the same commands given in Scenario 1
+above to install andsync your apps. Done!
+
+As you get more comfortable with Mackup and want to add new
+applications to sync, simply add the name of the supported application
+to the `.mackup.cfg` to the end of the file on a new line by itself.
```

Comment:
Can you specify how? (`mackup list`)

Reply:
Just give a hint for your readers to know how they can know what to add there, telling them to run `mackup list` and to copy past an app name from the list.

Reply:
Oh OK. Sure, will take care of this when I get a chance.